kinesumer.go

```diff
@@ -638,10 +640,19 @@ func (k *Kinesumer) Refresh(streams []string) {
 }
 
 // Errors returns error channel.
+// you must read from the Errors() channel or Kinesumer will ignore error if channel is blocked.
```

Comment:
Client don't HAVE TO read errors from it. It is optional. (but, of course it is not recommended.) So we don't need this comment line.